nucypher/policy/conditions/lingo.py

```diff
@@ -315,6 +318,10 @@ def eval(self, data) -> bool:
         result = _COMPARATOR_FUNCTIONS[self.comparator](left_operand, right_operand)
         return result
 
+    def with_resolved_context(self, **context):
```

Comment:
Should we add some unit tests in `test_return_value` for this function?

Reply:
✅